posts/2021/07/moving-scipy-to-meson.md

```diff
@@ -0,0 +1,489 @@
+<!--
+.. title: Moving SciPy to the Meson build system
+.. slug: moving-scipy-to-meson
+.. date: 2021-07-25 09:00:00 UTC-00:00
+.. author: Ralf Gommers
+.. tags: Labs, SciPy, Meson, build, packaging
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+Let's start with an announcement: _**SciPy now builds with
+[Meson](https://mesonbuild.com/) on Linux, and the full test suite passes!**_
+
+This is a pretty exciting milestone, and good news for SciPy maintainers and
+contributors - they can look forward to much faster builds and a more
+pleasant development experience. So how fast is it? Currently the build takes
+about 1min 50s on my 3 year old 12-core Intel CPU (i9-7920X @ 2.90GHz):
```

Comment:
Can you add something like "instead of more than 10 minutes when using the current `distutils` based build"

Reply:
adding "a ~4x improvement"